### PR TITLE
[SYCL] Align test result after uplift gpu driver 22.05.22297

### PR DESCRIPTION
**Pull request by yanfeng3721**

### CODE REVIEW

File: SYCL/Regression/static-buffer-dtor.cpp [hunk 1]
```diff
@@ -16,7 +16,7 @@
 // RUN: %ACC_RUN_PLACEHOLDER %t.out
 
 // Failing on HIP AMD
-// XFAIL: hip_amd
+// XFAIL: hip_amd || gpu
```
Comment (bader) [inline SYCL/Regression/static-buffer-dtor.cpp]: Won't it fail on Windows?

Comment (yanfeng3721) [reply SYCL/Regression/static-buffer-dtor.cpp]: Yes, I will modify it.

**Review by bader — CHANGES_REQUESTED** [review]
I don't think it fixes CUDA pre-commit.
I suggest we use `UNSUPPORTED: (level_zero || opencl) && linux`

**Comment by vladimirlaz** [comment]
The https://github.com/intel/llvm-test-suite/pull/871 should be used instead.

**Pull request closed without merging**